source/expressions.tex

```diff
@@ -6584,19 +6584,21 @@
 A \grammarterm{throw-expression} is of type \tcode{void}.
 
 \pnum
-Evaluating a \grammarterm{throw-expression} with an operand throws an
-exception\iref{except.throw}; the type of the exception object is determined by removing
-any top-level \grammarterm{cv-qualifier}{s} from the static type of the
-operand and adjusting the type
-from ``array of \tcode{T}'' or function type \tcode{T}
-to ``pointer to \tcode{T}''.
+A \grammarterm{throw-expression} with an operand throws an exception\iref{except.throw}.
+The array-to-pointer\iref{conv.array} and function-to-pointer\iref{conv.func}
+standard conversions are performed on the operand.
+The type of the exception object\iref{except.throw} is determined by
+removing any top-level cv-qualifiers from the type of the (possibly-converted) operand.
```

Comment:
This is technically a normative change: previously throwing a `const char[3]` would throw a `char*` rather than ` const char*`. But the old wording is sufficiently obviously wrong that that seems OK.

Reply:
Ah, I see that's #3959. I even managed to pick exactly the same example!